packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -334,6 +334,12 @@ class HotRunner extends ResidentRunner {
     }
 
     final Stopwatch restartTimer = new Stopwatch()..start();
+    if (previewDart2)
```

Comment:
Not sure about the style guide but feel the code will be more readable in this case if you added '{' '}' for this if statement.

Reply:
You are right, https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-braces-for-one-line-long-control-structure-statements talks about avoiding curly braces only around one-line long control structure statements.